.github/dependabot.yml

```diff
@@ -9,3 +9,504 @@ updates:
   directory: "/"
   schedule:
     interval: daily
+
+
+  - package-ecosystem: maven
+    directory: /m2e-maven-runtime/org.eclipse.m2e.archetype.common
+    schedule:
+      interval: daily
+
+  - package-ecosystem: maven
+    directory: /m2e-maven-runtime/org.eclipse.m2e.maven.runtime
+    schedule:
+      interval: daily
+
+  - package-ecosystem: maven
+    directory: /m2e-maven-runtime
+    schedule:
+      interval: daily
+
+  - package-ecosystem: maven
+    directory: /m2e-parent
+    schedule:
+      interval: daily
+
+  - package-ecosystem: maven
+    directory: /org.eclipse.m2e.apt.tests/projects/argumentMap
```

Comment:
I'm not sure  if dependencies in test resources should be updated.

Reply:
noted, thanks